pages/articles.tsx

```diff
@@ -28,6 +33,26 @@ export default function Articles({ tags, posts }) {
             ))
             .reduce((prev, curr) => [prev, ", ", curr])}
         </p>
+        <div className="stork">
```

Comment:
I think if you wanted, you can apply all this to the inner `stork-wrapper` div. That div doesn't do anything special (it's just for the theme), so you can hide/show it all you want!

Reply:
Neat idea, fixed.